src/chakra-components/control.tsx

```diff
@@ -98,7 +98,7 @@ const Control = <
       data-focus-visible={isFocused ? true : undefined}
       data-invalid={isInvalid ? true : undefined}
       data-disabled={isDisabled ? true : undefined}
-      aria-readonly={isReadOnly ? true : undefined}
+      data-readonly={isReadOnly ? true : undefined}
```

Comment:
This is not actually an input component, and does not have an aria role for an input, so it makes more sense to use the `data-readonly` prop, as this is just to trigger the readonly input styles from the Charka theme